Components/Payload/Providers/ShopperInfoProvider.php

```diff
@@ -31,7 +31,7 @@ public function provide(PaymentContext $context): array
             'billingAddress' => [
                 'city' => $context->getOrder()->getBilling()->getCity(),
                 'country' => $context->getOrder()->getBilling()->getCountry()->getIso(),
-                'houseNumberOrName' => $context->getOrder()->getBilling()->getNumber(),
+                'houseNumberOrName' => '',
```

Comment:
is it always an empty string if I understand correctly?

Reply:
@cyattilakiss Yes, this is the conversation in the PR https://github.com/Adyen/adyen-shopware5/pull/33

Reply:
```suggestion
                'houseNumberOrName' => 'N/A',
```